resources/readings/Testing.md

```diff
@@ -79,15 +79,15 @@ Testing as an engineering discipline notes
 
 * Steve Easterbrook [testing 2](http://www.cs.toronto.edu/~sme/CSC302/notes/17-Testing2.pdf).
 
-* Laurie Williams [black box testing notes](http://agile.csc.ncsu.edu/SEMaterials/BlackBox.pdf).
+* Laurie Williams [black box testing notes](https://students.cs.byu.edu/~cs340ta/spring2019/readings/BlackBox.pdf).
 
-* Laurie Williams [white box testing notes](http://agile.csc.ncsu.edu/SEMaterials/WhiteBox.pdf).
+* Laurie Williams [white box testing notes](https://students.cs.byu.edu/~cs340ta/spring2019/readings/WhiteBox.pdf).
 
 * Laurie Williams [automated testing notes](http://agile.csc.ncsu.edu/SEMaterials/AgileTesting.pdf) 
 
 * MIT testing [slides](http://measure.cc.gt.atl.ga.us/classes/cs6300/Lectures/class18testing.pdf).
```

Comment:
Can't find an alternate for this one, either. I also think this is a Georgia Tech course, not one by MIT.